scripts/publish.sh

```diff
@@ -5,11 +5,16 @@ cd ${OKTA_HOME}/${REPO}/packages/@okta/vuepress-site
 DEPLOY_ENVIRONMENT=""
 export REGISTRY_REPO="npm-topic"
 export REGISTRY="${ARTIFACTORY_URL}/api/npm/${REGISTRY_REPO}"
+export TREX_ENV=""
 
 declare -A branch_environment_map
 branch_environment_map[master]=vuepress-site-prod
 branch_environment_map[staging]=vuepress-site-preprod
 
+if [[ $BRANCH == "staging" ]]; then
+    TREX_ENV = "true"
+fi
+
```

Comment:
I don't think we need this variable since we will rely on `PROD_ENV` from the other PR